adafruit_circuitplayground/circuit_playground_base.py

```diff
@@ -518,11 +517,30 @@ def adjust_touch_threshold(self, adjustment):
               if cp.touch_A1:
                   print('Touched pad A1')
         """
-        for touch_in in self._touches:
+        for touch_in in self._touches.values():
             if isinstance(touch_in, touchio.TouchIn):
                 touch_in.threshold += adjustment
         self._touch_threshold_adjustment += adjustment
 
+    def _get_active_touchpad_items(self):
+        return [
+            (pinname, touchpad)
+            for pinname, touchpad in self._touches.items()
+            if isinstance(touchpad, touchio.TouchIn)
+        ]
+
```

Comment:
Can delete now with the changes above.
```suggestion
    def _get_active_touchpad_items(self):
        return [
            (pinname, touchpad)
            for pinname, touchpad in self._touches.items()
            if isinstance(touchpad, touchio.TouchIn)
        ]

```